modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library_admin/configuration_browse.jsp

```diff
@@ -0,0 +1,145 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+DLConfigurationDisplaycontext dlConfigurationDisplaycontext = new DLConfigurationDisplaycontext(request, renderRequest, renderResponse);
+DLGroupServiceSettings dlGroupServiceSettings = DLGroupServiceSettings.getInstance(scopeGroupId, request.getParameterMap());
+
+portletDisplay.setShowBackIcon(true);
+portletDisplay.setURLBack(dlConfigurationDisplaycontext.getBackURL());
+portletDisplay.setURLBackTitle("documents-and-media");
```

Comment:
Hi @ambrinchaudhary 
Yes, in a first attempt I tried to use the portletDisplay.getPortletDisplayName() but in none of these cases it would work because it returns Widget Configuration for all cases. So as in other sites in the portal, I decided that the easiest thing to do is to add it as language key 😄

Reply:
That is strange, but ok. If @adolfopa haven't complained, I'm ok with it ;-). Forwarding then...